pkg/trace/sampler/coresampler.go

```diff
@@ -32,10 +32,14 @@ type Sampler struct {
 	// seen counts seen signatures. In the case of the PrioritySampler, chunks dropped
 	// in the Client are also taken in account.
 	seen map[Signature][numBuckets]float32
+	// allSigsSeen counts all signatures
+	allSigsSeen [numBuckets]float32
```

Comment:
Can you please explain what these maps do and how they work (what the keys are and how they relate to the values). For both this and `seen`